core/storage/base_model/gae_models.py

```diff
@@ -1215,8 +1216,7 @@ def get_version(cls, entity_id, version_number, strict=True):
         except cls.EntityNotFoundError:
             if not strict:
                 return None
-            python_utils.reraise_exception() # type: ignore[no-untyped-call]
-            return None
+            return cast(None, python_utils.reraise_exception()) # type: ignore[no-untyped-call]
```

Comment:
Why are we changing this? The two statements seemed unrelated before.

Reply:
Moved the discussion to the doc.

Reply:
I have replied in the doc.

Reply:
Changed the implementation and used `# type: ignore[return]` as per the discussion on doc.